x-pack/plugins/siem/server/plugin.ts

```diff
@@ -169,11 +171,12 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       },
     });
 
-    if (plugins.alerting != null) {
+    if (plugins.alerting != null && plugins.ml != null) {
```

Comment:
Did you mean to check lists here instead?
```suggestion
    if (plugins.alerting != null && plugins.lists != null) {
```

Reply:
Yes thank you!